install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -5,43 +5,61 @@
 schema: |-
   definition user {}
 
+  // There's only one global installation
+  definition installation {
+
+    // only users that are not owned by an org are considered installation-level users
+    relation user: user
+    relation admin: user
+    permission is_admin = admin
+
+    // orgs can only be created by installation-level users
+    permission create_organization = user
+
+  }
+
   definition organization {
+    // Every organization is associated with the installation
+    relation installation: installation
+
     // Every user in an organization is automatically a member
     relation member: user
+    permission is_member = member + is_owner
     // Some users in an organization may additionally have the `owner` role
-    // granting them more privilidges with respect to the organization.
+    // granting them more privileges with respect to the organization.
     relation owner: user
+    permission is_owner = owner + installation->is_admin
```

Comment:
I find this a bit misleading, and would prefer that we actually are explicit with `installation->is_admin + owner` in the permissions.

The reason for the misleading part is that an admin is not an owner, and should never actually show up in the list of Owners for an organization

Reply:
Maybe renaming to `is_owner_or_admin` would be better, but if we ever extend this further, we can't keep renaming the permission

Reply:
Alright, gero had s similar desire to be explicit. Will propose a new version later.

Reply:
This may, however be complicated for the `project_edit` role as you'll need a synthetic permission anyway to avoid having to tie an installation to a project explicitly

Reply:
I've pushed a new proposal